lib/compress/zstd_double_fast.c

```diff
@@ -312,8 +359,12 @@ size_t ZSTD_compressBlock_doubleFast_dictMatchState_generic(
         U32 offset;
         size_t const h2 = ZSTD_hashPtr(ip, hBitsL, 8);
         size_t const h = ZSTD_hashPtr(ip, hBitsS, mls);
-        size_t const dictHL = ZSTD_hashPtr(ip, dictHBitsL, 8);
-        size_t const dictHS = ZSTD_hashPtr(ip, dictHBitsS, mls);
+        size_t const dictHashAndTagL = ZSTD_hashPtr(ip, dictHBitsL + ZSTD_SHORT_CACHE_TAG_BITS, 8);
+        size_t const dictHashAndTagS = ZSTD_hashPtr(ip, dictHBitsS + ZSTD_SHORT_CACHE_TAG_BITS, mls);
```

Comment:
Also same kind of comment. Would it be cleaner overall to do the `+ TAG_BITS` adjustment when we initialize these variables rather than at every use?

Reply:
Sure, I can add the tag bits when initializing `dictHBitsL`.